### ISSUE TITLE
Find a to way to compile Arduino targeting windows 

### ISSUE THREAD

**Issue by MatthewColvin** [CoretechR/OMOTE#23]
This would allow us to put Arduino library dependencies in the OmoteUI.hpp which would be really nice.

**Comment by Dark1886** [comment]
Can you explain the exact issue here? I would like to assist but seem a little confused about the actual problem.

**Comment by MatthewColvin** [comment]
I don't think this original ticket was feasible but this accomplishes a similar task. 

Thanks to @Dark1886 for pointing me to https://github.com/lvgl/lv_platformio I have an extremely rough version of the simulator working in platform IO so hopefully we can ditch the solution after we get it properly added :) 

CC: @CoretechR
